sn_collections/_guides/web-based-login.md

```diff
@@ -15,48 +18,62 @@ GUIs:
   - title: Open OnDemand Interface
     url: /guides/software/open-ondemand
     internal: true
-    shows_list: subnav
   - title: Galaxy on SCINet
     url: /guides/software/galaxy
     internal: true
-    shows_list: subnav
+  - title: Globus Online
+    url: /guides/data/datatransfer#globus-data-transfer
+    excerpt: "Globus Online is the recommended method for transferring data to and from the Ceres cluster. It provides faster data transfer speeds compared to scp, has a graphical interface, and does not require a GA verification code for every file transfer. To transfer data to/from a local computer, users will need to install Globus Connect Personal which does NOT require admin privileges."
```

Comment:
"Globus Online is the recommended method for transferring data to or among Ceres, Atlas, and Juno. It provides faster data transfer speeds compared to scp, has a graphical interface, and does not require authentication for every file transfer. To transfer data to or from a local computer, users will need to install Globus Connect Personal which does NOT require admin privileges."